core/script.runtime/src/kotlin/script/dependencies/dependencies.kt

```diff
@@ -20,6 +20,15 @@ package kotlin.script.dependencies
 
 import java.io.File
 
+// discuss
+
+// Currently File is used to disambiguate between plain strings and paths
+// One of the ideas is to use java.nio.Path but this requires JDK 7
+// that means that script templates would require higher JDK (but since script are run by calling koltinc it seems ok to me after consideration)
```

Comment:
Perhaps yes. The environment in which we'd compile script using e.g. the daemon running under java 8, but run it under java 6 is theoretically possible but practically seems too complicated to be useful.